muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/note-book/hooks/useScroll.ts

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+
+/**
+ * IUseScroll
+ */
+export interface UseScroll {
```

Comment:
documentation says IUseScroll and interface is UseScroll, which is the same name as the hook, which is a problem?